dotnet-desktop-guide/net/winforms/whats-new/net60.md

```diff
@@ -0,0 +1,246 @@
+        Application.Run(new Form1());
+    }
+}
+```
+
+In .NET 6, these templates have been modified to use the new bootstrap code, invoked by the `ApplicationConfiguration.Initialize` method.
+
+```csharp
+class Program
+{
+    [STAThread]
+    static void Main()
+    {
+        ApplicationConfiguration.Initialize();
+        Application.Run(new Form1());
+    }
+}
+```
+
+This method is automatically generated at compile time and contains the code to configure Windows Forms. The project file can control these settings now too, and you can avoid configuring it in code. For example, the generated method looks similar to the following code:
+
+```csharp
+public static void Initialize()
+{
+    Application.EnableVisualStyles();
+    Application.SetCompatibleTextRenderingDefault(false);
+    Application.SetHighDpiMode(HighDpiMode.SystemAware);
+}
+```
+
+The new bootstrap code is used by Visual Studio to configure the Windows Forms Visual Designer. If you opt-out of using the new bootstrap code, by restoring the old code and bypassing the `ApplicationConfiguration.Initialize` method, the Windows Forms Visual Designer won't respect the bootstrap settings you set.
+
+The settings generated in the `Initialize` method are controlled by the project file.
+
+### Project-level application settings
+
+To complement the [new application bootstrap](#new-application-bootstrap) feature of Windows Forms, a few `Application` settings previously set in the startup code of the application should be set in the project file. The project file can configure the following application settings:
+
+| Project setting                                                                                                               | Default value | API                                                                                    |
+|-------------------------------------------------------------------------------------------------------------------------------|-- |----------------------------------------------------------------------------------------|
+| [ApplicationVisualStyles](/dotnet/core/project-sdk/msbuild-props-desktop#applicationvisualstyles)                             | `true` | `Application.EnableVisualStyles();`                                                    |
+| [ApplicationUseCompatibleTextRendering](/dotnet/core/project-sdk/msbuild-props-desktop#applicationusecompatibletextrendering) | `false` | `Application.SetCompatibleTextRenderingDefault(false);`                                |
+| [ApplicationHighDpiMode](/dotnet/core/project-sdk/msbuild-props-desktop#applicationhighdpimode)                               | `SystemAware` | `Application.SetHighDpiMode(HighDpiMode.SystemAware);`                                 |
+| [ApplicationDefaultFont](/dotnet/core/project-sdk/msbuild-props-desktop#applicationdefaultfont)                               | `Segoe UI, 9pt` | `Application.SetDefaultFont(new Font(new FontFamily("Microsoft Sans Serif"), 8.25f));` |
```

Comment:
This looks strange:
![image](https://user-images.githubusercontent.com/4403806/141384691-38359dd3-d9d0-43d4-917e-a996e4b8b9c4.png)

Should this be:
```suggestion
| [ApplicationDefaultFont](/dotnet/core/project-sdk/msbuild-props-desktop#applicationdefaultfont)                               | `Microsoft Sans Serif, 8.25f` | `Application.SetDefaultFont(new Font(new FontFamily("Microsoft Sans Serif"), 8.25f));` |
```

Ditto below:
```
<ApplicationDefaultFont>Microsoft Sans Serif, 8.25f</ApplicationDefaultFont>
```

Reply:
I think this should actually be
```suggestion
| [ApplicationDefaultFont](/dotnet/core/project-sdk/msbuild-props-desktop#applicationdefaultfont)                               | `Segoe UI, 9pt` | `Application.SetDefaultFont(new Font(new FontFamily("Segoe UI"), 9f));` |
```